_mentalModels/2022-03-20-stairs.md

```diff
@@ -0,0 +1,61 @@
+---
+title: The Stairs
+
+draft: false
+
+page : false 
+
+image: /assets/img/mental-models/stairs.svg
+
+excerpt: >
+  The Stairs is a good metaphor to think about _progression_- specially with a human factor.
+
+---
+
+<img
+  alt="A schematic stairs pattern"
+  src="/assets/img/mental-models/stairs.svg"
+  style="max-height: 480px; margin: 2rem auto"
+/>
+
+The Stairs is a good metaphor to think about _progression_ - specially with a
+human factor.
+
+It could be about the way a team project is going at work, or the typical
+learning curve of mastering a new language. It can even be about one's personal
+advancement toward a career goal.
+
+It could looks like linear at first thought. We have a starting line and a goal.
+Isn't the progression like the straight line between the two? It's certainly
+not. Taking a step back for the progression itself helps us understand the real
+pattern.
+
+This is never linear. We alternates between moments of slow and quick
+progression. It's like going up a stair with very long step.
+
+<img
+  alt="A schematic stairs with two tiny persons. One is walking. One is climbing a step"
+  src="/assets/img/mental-models/stairs-people.svg"
+  style="max-height: 480px; margin: 2rem auto"
+/>
+
+There are moments where we're walking peacefully on one step. We're not making
+much visible progress but we're certainly progressing towards the next step.
+We're consolidating previous advancements, accumulating experience at a
+sustainable pace.
+
+There are moments where we surprise ourself climbing a step. We have the
+sensation of having made deep progress in a very short time. A fundamental
+concept just clicked. The team delivered a important component of the project.
+Personal and company goals finally compound.
+
+We alternate between these two very different kinds of progression moments.
+
+---
+
+Asking ourself where we are on The Stairs is important. Are we "climbing a
+step"? Then dedicating a bit more energy into it could help giving it a helpful
+push.
```

Comment:
```suggestion
Asking ourselves where we currently are on The Stairs is important. Are we "climbing a
step"? Then dedicating a bit more energy into it could help us reach a new and valuable higher floor.
```